Expensify/App
# Rename profile page to details page

### Details
Currently in E.cash there are two profile pages: The page you get when you click on the chat header avatar of a chat you have with someone, and the page you get when navigating to Settings > Profile. 

The non-settings profile page displays a header of `Details` while the settings profile page displays `Profile`. 

There's no reason for these pages to be named the same thing, and from the description above you can see how much more difficult to describe which page is which. 

### Fixed Issues
Fixes N/A, was brought up in conversation and is an easy fix. 

### Tests
1. Log into E.cash 
2. Create a chat with one person 
3. Click on their chat header avatar and confirm you can see the `Details` page for that user. 

### Tested On


### Screenshots
#### Web
<img width="1200" alt="Screen Shot 2021-03-19 at 10 59 21 AM" src="https://user-images.githubusercontent.com/16747822/111823417-2e6e6b80-88a2-11eb-911f-614d096e2628.png">

#### Mobile Web
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111826054-75aa2b80-88a5-11eb-99a1-fc1923bb0a2e.png">

#### Desktop
<img width="900" alt="Screen Shot 2021-03-19 at 11 00 15 AM" src="https://user-images.githubusercontent.com/16747822/111825334-9625b600-88a4-11eb-9a30-c0eccd1d92ef.png">

#### iOS
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111825719-06343c00-88a5-11eb-9eb6-d8b9782363d0.png">

#### Android
<img width="200" alt="" src="https://user-images.githubusercontent.com/16747822/111825401-aa69b300-88a4-11eb-9175-5882f4116733.png">
